<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BagpipeAssistant</class>
 <widget class="QWidget" name="BagpipeAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>717</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>255</width>
    <height>205</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#BagpipePanel {
background-color: transparent
}
#panelToolBox {
background-color: transparent
}
#graceNoteDurationsPage {
background-color: rgba(0,0,0,0)
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="panelToolBox">
     <property name="toolTip">
      <string>Modifies all embellishment note durations to defined lengths so similar embellishments sound the same and do not vary based on their parent note's duration.</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="embellishmentDurationsPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>645</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Embellishment Durations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="modifyDurationsPushButton">
         <property name="text">
          <string>Modify Durations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="durationsGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Durations</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QSpinBox" name="baseDurationSpinBox">
            <property name="suffix">
             <string> ms</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Base Duration</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="tunePropertiesGroupBox">
         <property name="title">
          <string>Tune Properties</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="tuneTypeLabel">
            <property name="text">
             <string>Tune Type</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="tuneTypeComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="goalTempoLabel">
            <property name="text">
             <string>Goal Tempo</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="goalTempoSpinBox">
            <property name="suffix">
             <string> bpm</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="singleStep">
             <number>4</number>
            </property>
            <property name="value">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="practiceTempoLabel">
            <property name="text">
             <string>Practice Tempo</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="practiceTempoSpinBox">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="stretchEmbellishmentsSpinBox">
            <property name="toolTip">
             <string>Additional duration applied to each embellishment note.</string>
            </property>
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Stretch Embellishments</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="dontStretchCutsCheckBox">
            <property name="toolTip">
             <string>If checked, and practice tempo is less than 100%, cut note durations will be set based on goal tempo, and corresponding dotted notes will be lengthened accordingly.</string>
            </property>
            <property name="text">
             <string>Don't Stretch Cut Notes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="embellishmentPositionGroupBox">
         <property name="title">
          <string>Embellishment Position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="beforeNoteStartRadioButton">
            <property name="text">
             <string>Before Start of Note</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="variesNoteStartRadioButton">
            <property name="text">
             <string>Varies (Defined by embellishment)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="afterNoteStartRadioButton">
            <property name="text">
             <string>After Start of Note</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="beatStressPatternGroupBox">
         <property name="title">
          <string>Beat Stress Pattern</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QComboBox" name="beatStress1ComboBox">
            <property name="frame">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="beatStress2ComboBox"/>
          </item>
          <item>
           <widget class="QComboBox" name="beatStress3ComboBox"/>
          </item>
          <item>
           <widget class="QComboBox" name="beatStress4ComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="modifyNoteDurationPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>249</width>
        <height>645</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Canntaireachd</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>modifyDurationsPushButton</tabstop>
  <tabstop>tuneTypeComboBox</tabstop>
  <tabstop>goalTempoSpinBox</tabstop>
  <tabstop>practiceTempoSpinBox</tabstop>
  <tabstop>dontStretchCutsCheckBox</tabstop>
  <tabstop>stretchEmbellishmentsSpinBox</tabstop>
  <tabstop>afterNoteStartRadioButton</tabstop>
  <tabstop>beatStressPatternGroupBox</tabstop>
  <tabstop>beatStress1ComboBox</tabstop>
  <tabstop>beatStress2ComboBox</tabstop>
  <tabstop>beatStress3ComboBox</tabstop>
  <tabstop>beatStress4ComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
